torch/csrc/jit/codegen/cuda/runtime/fused_welford_impl.cu

```diff
@@ -442,6 +443,79 @@ __device__ __inline__ void ParallelReduce<
   block_sync::sync();
 }
 
+template <
+    int X_BLOCK,
+    int Y_BLOCK,
+    int Z_BLOCK,
+    int X_THREAD,
+    int Y_THREAD,
+    int Z_THREAD,
+    bool PERSISTENT_REDUCTION,
+    bool BROADCAST>
+template <int NumArgs, typename DataType, typename IndexType>
+__device__ __inline__ void ParallelReduce<
+    X_BLOCK,
+    Y_BLOCK,
+    Z_BLOCK,
+    X_THREAD,
+    Y_THREAD,
+    Z_THREAD,
+    PERSISTENT_REDUCTION,
+    BROADCAST>::
+    welfordGroup(
+        typename MakeRefTuple<NumArgs, DataType>::type out_avg,
+        typename MakeRefTuple<NumArgs, DataType>::type out_var,
+        typename MakeRefTuple<NumArgs, IndexType>::type out_N,
+        const typename MakeConstRefTuple<NumArgs, DataType>::type& inp_avg,
+        const typename MakeConstRefTuple<NumArgs, DataType>::type& inp_var,
+        const typename MakeConstRefTuple<NumArgs, IndexType>::type& inp_N,
+        const typename MakeLocalTuple<NumArgs, DataType>::type& init_avg,
+        const typename MakeLocalTuple<NumArgs, DataType>::type& init_var,
+        const typename MakeLocalTuple<NumArgs, IndexType>::type& init_N,
+        typename MakeVolatilePtrTuple<NumArgs, DataType>::type
+            global_work_buffer_avg,
+        typename MakeVolatilePtrTuple<NumArgs, DataType>::type
+            global_work_buffer_var,
+        typename MakeVolatilePtrTuple<NumArgs, IndexType>::type
+            global_work_buffer_N,
+        int64_t* global_sync_buffer,
+        PtrTuple<DataType, DataType, IndexType> shared_buf,
+        const typename MakeLocalTuple<NumArgs, bool>::type& read_preds,
+        const typename MakeLocalTuple<NumArgs, bool>::type& write_preds,
+        int64_t& cycles,
```

Comment:
What's cycles and count used for?

Reply:
They are for a quick kernel profiling with `PYTORCH_NVFUSER_ENABLE=kernel_profile`. Each `Kernel` has `KernelPerformanceProfile` as defined at https://github.com/csarofeen/pytorch/blob/devel/torch/csrc/jit/codegen/cuda/kernel.h#L101.